lib/sedna/README.md

```diff
@@ -1,2 +1,285 @@
+|   |       |-- inference_integrate.py  # Integrating algorithm for the output geted by multitask inference
+|   |       |-- task_definition.py      # Multitask definition base on given traning samples
+|   |       |-- task_mining.py  # Mining target tasks of inference samples
+|   |       |-- task_relation_discover.py  # Discover the relation of tasks which generated from task_definition
+|   |       `-- task_remodeling.py  # Remodeling tasks
+|   `-- unseen_task_detect
+|       |-- __init__.py
+|       `-- unseen_task_detect.py      # Unseen task detect algorithms for lifelong learning
+|-- backend                     # Encapsulated the general ML frameworks, decoupled the core algorithms of sedna from the framework
+|   |-- __init__.py
+|   |-- base.py
+|   `-- tensorflow
+|       `-- __init__.py
+|-- common                     # Contains the common methods, configurations and class factory
+|   |-- __init__.py
+|   |-- class_factory.py       # It defines two importantt classes, ClassType and ClassFactory. You can register the modules which want to be reuse through the decorator.
+|   |-- config.py              # Globally used configuration information
+|   |-- constant.py            # 
+|   |-- file_ops.py            # File system processing 
+|   |-- log.py                 # Global log definition 
+|   `-- utils.py               # Common funtions
+|-- core                       # All features build in the sedna framework
+|   |-- __init__.py
+|   |-- base.py                # Abstract Base class of the features
+|   |-- federated_learning     # Federated learning
+|   |   |-- __init__.py
+|   |   `-- federated_learning.py
+|   |-- incremental_learning   # Incremental learning
+|   |   |-- __init__.py
+|   |   `-- incremental_learning.py
+|   |-- joint_inference        # Joint inference
+|   |   |-- __init__.py
+|   |   `-- joint_inference.py
+|   `-- lifelong_learning      # Lifelong learning
+|       |-- __init__.py
+|       `-- lifelong_learning.py
+|-- datasources                # Abastract Base class of the dataset (train/eval/test)
+|   `-- __init__.py
+`-- service                    # Communication module, include clinet/server
+    |-- __init__.py
+    |-- client.py              # Send http/ws requests to the server 
+    |-- run_kb.py              # Run Knowledgebase Service as rest api
+    `-- server                 # Servers for each feature
+        |-- __init__.py 
+        |-- aggregation.py     # Aggregator service for federated learning
+        |-- base.py            # Base module
+        |-- inference.py       # Inference Service for Joint inference
+        `-- knowledgeBase      # Knoledgebase Service
+            |-- __init__.py
+            |-- database.py
+            |-- model.py       # Knoledgebase table ORM
+            `-- server.py
+```
+
+## Customize algorithm
+
+Sedna provide a class called `class_factory.py` in `common` package, only a few lines of changes are required to become a module of sedna.
```

Comment:
1、provide -> provides
2、"and only a few..."

Reply:
fix